doc/lsst.cp.pipe/constructing-calibrations.rst

```diff
@@ -0,0 +1,733 @@
+        u/czw/DM-28920/calib.20210720 \
+        u/czw/DM-28920/calib/flat.20210720
+
+- With the flat created, the defects can be reconstructed using both bias and flat images:
+.. code:: bash
+
+    butler collection-chain /repo/main --mode=remove \
+        u/czw/DM-28920/calib.20210720 \
+        u/czw/DM-28920/calib/defect.20210720
+    butler certify-calibrations /repo/main \
+        u/czw/DM-28920/defectGen.20210720a \
+        u/czw/DM-28920/calib/defect.20210720a \
+        --begin-date 2020-01-01 --end-date 2050-01-01 defects
+    butler collection-chain /repo/main --mode=extend \
+        u/czw/DM-28920/calib.20210720 \
+        u/czw/DM-28920/calib/defect.20210720a
+
+- The PTC is not generally used outside of calibration production, so the initial pass can be certified to a temporary collection:
+.. code:: bash
+
+    butler certify-calibrations /repo/main \
+        u/czw/DM-28920/ptcGen.20210721a \
+        u/czw/DM-28920/tempPtcA.0721 \
+        --begin-date 2019-01-01 --end-date 2050-01-01 ptc
+    butler certify-calibrations /repo/main \
+        u/czw/DM-28920/ptcGen.20210721b \
+        u/czw/DM-28920/tempPtcB.0721 \
+        --begin-date 2019-01-01 --end-date 2050-01-01 ptc
+
+- That PTC can be used to construct a linearity solution:
+.. code:: bash
+
+    butler certify-calibrations /repo/main \
+        u/czw/DM-28920/linearityGen.20210721Xa \
+        u/czw/DM-28920/calib/linearity.20210721 \
+        --begin-date 2020-01-01 --end-date 2050-01-01 linearity
+    butler collection-chain /repo/main --mode=extend \
+        u/czw/DM-28920/calib.20210720 \
+        u/czw/DM-28920/calib/linearity.20210721
+
+- Which can be used to update the PTC and remove linearity effects:
+.. code:: bash
+
+    butler certify-calibrations /repo/main\
+        u/czw/DM-28920/ptcGen.20210721Ya \
+        u/czw/DM-28920/ptcA.20210721 \
+        --begin-date 2019-01-01 --end-date 2050-01-01 ptc
+    butler certify-calibrations /repo/main
+        u/czw/DM-28920/ptcGen.20210721Yb \
+        u/czw/DM-28920/ptcB.20210721 \
+        --begin-date 2019-01-01 --end-date 2050-01-01 ptc
+
+- The updated PTC can be used to create a brighter-fatter kernel:
+.. code:: bash
+
+    butler certify-calibrations /repo/main \
+        u/czw/DM-28920/bfkGen.20210721a \
+        u/czw/DM-28920/bfk.20210721 \
+        --begin-date 2020-01-01 --end-date 2050-01-01 bfk
+    butler collection-chain /repo/main --mode=extend \
+        u/czw/DM-28920/calib.20210720 \
+        u/czw/DM-28920/calib/bfk.20210721
+
+- With a full set of calibrations, the crosstalk can be measured:
+.. code:: bash
+
+   butler certify-calibrations /repo/main \
+       u/czw/DM-28920/crosstalkGen.20210721a \
+       u/czw/DM-28920/crosstalk.20210721
+       --begin-date 2020-01-01 --end-date 2050-01-01 crosstalk
+   butler collection-chain /repo/main --mode=extend \
+       u/czw/DM-28920/calib.20210720 u/czw/DM-28920/calib/crosstalk.20210721
+
+.. _cp-pipe-readNoise:
+
+Read Noise
```

Comment:
I've grouped all of the "construction" parts into a new top-level section.

I've also added a link to `ip_isr`, although that is also lacking documentation, so that's likely a future ticket.